viam-cartographer.go

```diff
@@ -177,6 +186,27 @@ func New(
 	return cartoSvc, nil
 }
 
+func setUpCartoFacadeQueue(cartoFacadeCancelCtx context.Context, activeBackgroundWorkers *sync.WaitGroup) cfq.CartoFacadeQueue {
```

Comment:
Lets move this into viam-cartographer/src/carto_facade_queue/carto_facade_queue.go & make it public